### ISSUE TITLE
require with a JPG image fails with an error message

### ISSUE THREAD

**Issue by ttraenkler** [electron/electron-compile#204]
Attempting to load an image with 

```
const wallpaper = require("../../../../../public/wallpaper.jpg")
```
in my react-native-web project using electron-prebuilt-compile fails with (I shortened the path but it is correct):

`null code returned for "/Users/{...}/public/wallpaper.jpg".  Please raise an issue on 'electron-compile' with the contents of this file.
`

The relative path is resolved correctly. What I have already tried without success: Importing the image with `import` at the top of the file instead of require and putting the file in the same directory as the source. I have also tried a different jpg image to make sure the specific jpg image file is not the problem. I was able to successfully load the image in the index.html as `background` image however (not using require).

**Comment by MarshallOfSound** [comment]
What are you expecting to happen when you require an image???

Webpack has loaders that do magical things with different file types, node / electron does not.  If you want to put an image on the page set it's src to be that path you provided 👍

**Comment by ttraenkler** [comment]
I tried a relative path string as well - however given your hint I realized the relative path that is used in react-native + react-native-web needs to be replaced with a path relative to the project directory: `const image = "./public/image.jpg"`

**Comment by danielo515** [comment]
> What are you expecting to happen when you require an image???

I expect the compiler to interpret it, include the image on the bundle and give the proper path to me.
I don't understand your surprise since this is quite common on react and you have a react template.
I looked for documentation on electron-forge about how to configure it to match the behavior I described and I failed. There is no specific section about how to handle assets either.
